CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the SwiftFoundation open source project
```

Comment:
I think we just use "Swift" not "SwiftFoundation" in our other copyright headers.